docs/about/local-environment-setup.rst

```diff
@@ -79,9 +79,7 @@ Install helm
 
 Some Phalanx commands require Helm (v3 or later) to be available on your PATH.
 Any version of Helm after v3 should be okay.
-You therefore must have it installed on your PATH.
-
-See the `Helm installation guide <https://helm.sh/docs/intro/install/>`__ for more details.
+See the `Helm installation guide <https://helm.sh/docs/intro/install/>`__ for installation instructions.
```

Comment:
Did people really not see "Helm installation guide" as being "for installation instructions?" I guess we can repeat the word "installation" twice in a sentence to make a point... 😂